src/interface/AuthResponse/AuthResponse.tsx

```diff
@@ -15,7 +15,7 @@ export interface AxiosErrorResponse {
     country: string;
     phoneNumber: string;
     confirmPassword: string;
-    profileImage: string | File | HTMLImageElement | null|Blob; // توضيح نوع الصورة بدلاً من استخدام `any`.
+    profileImage:any; 
```

Comment:
From the backend side, it will be just a string.

Reply:
done